text/0000-copy-clone-semantics.md

```diff
@@ -0,0 +1,56 @@
+- Feature Name: N/A
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+With specialization on the way, we need to talk about the semantics of
+`<T as Clone>::clone() where T: Copy`.
+
+It's generally been an unspoken rule of Rust that a `clone` of a `Copy` type is
```

Comment:
(The example of specializing `#[derive(Copy, Clone)]` that was brought up elsewhere in the thread also seems relevant.)